rom/dev/tests/test_image_validation.rs

```diff
@@ -341,6 +373,80 @@ fn test_header_verify_vendor_sig_mismatch() {
     );
 }
 
+#[test]
+fn test_header_verify_vendor_lms_sig_mismatch() {
+    let (mut hw, mut image_bundle) =
+        helpers::build_hw_model_and_image_bundle(Fuses::default(), ImageOptions::default());
```

Comment:
All these tests indicate default fusing.  How do we know that LMS has signed and verified?

Reply:
The check for the LMS verification fuse is not in this change, so by default, LMS verification is on. Once this change is checked in, I am planning to add the fuse checks.